common/app/model/EmailNewsletters.scala

```diff
@@ -39,8 +39,8 @@ object EmailNewsletters {
   val guardianTodayUs = EmailNewsletter(
     name = "Guardian Today",
     theme = "news",
-    teaser = "Our editors break down the big news for you in the daily briefing",
-    description = "Our editors' picks for the day's top news and commentary delivered to your inbox each morning.",
+    teaser = "Cut through the noise and get straight to the heart of the day's breaking news in double-quick time with our daily email.",
```

Comment:
This is being very picky so sorry 😉  but I think the `'` should be a `’` here (curly apostrophe rather than straight)